packages/muelu/src/Graph/MueLu_CoalesceDropFactory_def.hpp

```diff
@@ -828,7 +828,7 @@ namespace MueLu {
       GetOStream(Statistics1) << "CoalesceDropFactory: nodeMap " << nodeMap->getNodeNumElements() << "/" << nodeMap->getGlobalNumElements() << " elements" << std::endl;
 
       // 3) create graph of amalgamated matrix
-      RCP<CrsGraph> crsGraph = CrsGraphFactory::Build(nodeMap, 10, Xpetra::DynamicProfile);
+      RCP<CrsGraph> crsGraph = CrsGraphFactory::Build(nodeMap, 10, Xpetra::StaticProfile);
```

Comment:
@GeoffDanielson Ugh.  I suspect `10` is arbitrary, and that this code relies on having the matrix expand its row storage as needed.  Did you run the MueLu unit tests to see if this branch is ever hit?

Reply:
I did run the muelu unit tests and the limit doesn't get hit.  I suspect you're right, but I'm not sure whether it'd be more correct to fix it with the proper upper limit or just leave as is.

Reply:
If the matrix is an Xpetra::BlockedCrsMatrix, then the method `getNodeMaxNumRowEntries` is available.  I am fine with this change.